src/app/functions/universal/breach.ts

```diff
@@ -53,6 +75,17 @@ export const SecurityRecommendationDataTypes = {
   IP: BreachDataTypes.IP,
 } as const;
 
+export type HibpBreachDataTypes = typeof BreachDataTypes;
+export interface BreachBulkResolutionRequest {
+  dataType: HibpBreachDataTypes[keyof HibpBreachDataTypes];
+}
+
+export interface BreachResolutionRequest {
+  affectedEmail: string;
+  breachId: number;
+  resolutionsChecked: Array<HibpBreachDataTypes[keyof HibpBreachDataTypes]>;
+}
+
```

Comment:
Wouldn't it make more sense to move this to the API route that actually expects this request body?

Reply:
Makes sense to me — updated in [f884e04](https://github.com/mozilla/blurts-server/pull/4332/commits/f884e04dfe3a196d532650aad1785ac5c4d14254).